vignettes/adsl.Rmd

```diff
@@ -716,6 +717,36 @@ dataset_vignette(
 )
 ```
 
+### Derive Randomization Date (`RANDDT`) {#randomization_date}
+
+The function `derive_vars_merged_dt()` can be used to derive randomization date 
+variable.
+
+By default, the function will map DS records where DSDECOD="RANDOMIZED". To derive 
+the Randomization Date (`RANDDT`), the call would be:
+
+```{r eval=TRUE}
+adsl <- adsl %>%
+  derive_vars_merged_dt(
+    dataset_add = ds,
+    filter_add = DSDECOD == "RANDOMIZED",
+    by_vars = vars(STUDYID, USUBJID),
+    new_vars_prefix = "RAND",
+    dtc = DSSTDTC,
+    flag_imputation = FALSE
+  )
+```
+
+
+This call would return the input dataset with the column `RANDDT` and by default
+the associated date imputation flag (`RANDDTF`) are added. If the imputation flag
+is not required, the user must set the argument `flag_imputation` to FALSE.
```

Comment:
Delete "and by default the associated date imputation flag (`RANDDTF`) are" and delete "If the imputation flag
is not required, the user must set the argument `flag_imputation` to FALSE."

Reply:
If we do not set `flag_imputation = FALSE` then only this is not added, otherwise by default the function `derive_vars_merged_dt()` adds the imputation flag variable with `NA` values. Let me know your views.

Reply:
i think this is all due to the fact you're not using latest version of the function, but let me know if i'm mistaken